.github/workflows/pr-check.yaml

```
@@ -276,6 +276,7 @@ jobs:
       inference_model: 'granite3.2:2b'
       inference_second_model: 'granite3.1-dense:2b'
       compile_args: '--linux dir'
+      test_script: 'test:e2e:smoke'
```

actionlint (1.7.12)

[error] input "test_script" is not defined in "./.github/workflows/e2e-test-job.yaml" reusable workflow. defined inputs are "compile_args", "inference_model", "inference_provider", "inference_second_model", "instance", "mode", "ref", "repo"

(workflow-call)